accepted/2020/workloads/workload-manifest.md

```diff
@@ -218,17 +220,19 @@ For example, an SDK pack might contain a compiler that's a native executable and
 }
 ```
 
-Workload definitions and SDK imports would refer to this as `"foo.sdk.compiler"`, but on Windows the pack that gets installed and resolved would be `"foo.sdk.compiler.windows"`. The `"foo.sdk.compiler.windows"` pack would be present on disk and its ID would be visible in resolved paths, but otherwise transparent and not able to be used directly.
+Workload definitions and SDK imports would refer to this as `foo.sdk.compiler`, but on Windows the pack that gets installed and resolved would be `foo.sdk.compiler.windows`. The `foo.sdk.compiler.windows` pack would be present on disk and its ID would be visible in resolved paths, but otherwise transparent and not able to be used directly. On x64 Macs it would similarly resolve to the concrete `foo.sdk.compiler.mac` pack.
+
+On all other platforms this pack would be a no-op. If workloads that contained this pack were not functional without it, the workload author should restrict the workload itself using `"platforms": [ "osx-x64", "win-x64" ]`.
```

Comment:
What does it mean to be a no-op?  I think it means it would be a no-op on workload install, but what about during workload resolution?  Should the `workloadmanifest.targets` file have conditions that make sure the workload pack isn't imported on platforms where it's not available?

Reply:
Clarified this in the spec